charts/buildkit-service/values.yaml

```diff
@@ -72,3 +72,7 @@ autoscaling:
   targetCPUUtilizationPercentage: 80
   # -- Target memory utilization percentage
   targetMemoryUtilizationPercentage: 80
+
+# -- Lifecycle hooks and termination
```

Comment:
These values need to be added in the README as well. It is generated automatically by [helm-docs](https://github.com/norwoodj/helm-docs) but You can also update it manually. It shows what is missing in the failing docs job.